libtracefs will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libtracefs it will fail in the following way:

```
2024/11/12 23:12:30 INFO running step "test/pkgconf"
2024/11/12 23:12:30 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:30 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:30 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:30 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:30 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + basename /home/build/melange-out/libtracefs-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + dev_pkg=libtracefs-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + apk info -L libtracefs-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + grep -q ^Name: usr/lib/pkgconfig/libtracefs.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + basename usr/lib/pkgconfig/libtracefs.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + lib_name=libtracefs uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + echo usr/lib/pkgconfig/libtracefs.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + grep -q '^usr/lib/pkgconfig/libtracefs.pc$\|^usr/share/pkgconfig/libtracefs.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + pkgconf --exists libtracefs uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN + pkgconf --print-errors --exists libtracefs uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN Package libtraceevent was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN Perhaps you should add the directory containing `libtraceevent.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 WARN Package 'libtraceevent', required by 'libtracefs', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:30 INFO pod d3b686e778b78d6a0f72ee3e9925b88f0b804bfdda6c1ec7d79438abfd37e42e terminated
2024/11/12 23:12:30 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:30 INFO   workspace dir: /home/user/tmp/melange-workspace-3690187455
2024/11/12 23:12:30 INFO   guest dir: /home/user/tmp/melange-guest-2410091336
2024/11/12 23:12:30 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libtracefs] Error 1
```

--- a/libtracefs.yaml
+++ b/libtracefs.yaml
@@ -1,7 +1,7 @@
 package:
   name: libtracefs
   version: 1.8.1
-  epoch: 0
+  epoch: 1
   description: "Linux kernel trace file system library"
   copyright:
     - license: "LGPL-2.1-or-later"
@@ -33,7 +33,13 @@ subpackages:
   - name: libtracefs-dev
     pipeline:
       - uses: split/dev
+    dependencies:
+      runtime:
+        - libtraceevent-dev
     description: libtracefs dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

